models/layerzero/arbitrum/layerzero_arbitrum_send.sql

```diff
@@ -1,4 +1,5 @@
 {{ config(
+    tags=['dunesql'],
     alias = alias('send'),
     partition_by = ['block_date'],
```

Comment:
we've been looking to increase our partition size -- can you introduce a new column called `block_month`, which mimics `block_date` but truncates at the monthly level? then apply column to partition?

this applies to all blockchain version spells here

Reply:
Updated.